support/iosConstants.ts

```diff
@@ -1,2 +1,11 @@
 export const IPHONEOS_DEPLOYMENT_TARGET = "10.0";
 export const TARGETED_DEVICE_FAMILY = `\"1,2\"`;
+
+export const NSE_PODFILE_SNIPPET = `
+target 'OneSignalNotificationServiceExtension' do
+  pod 'OneSignal', '>= 3.0', '< 4.0'
```

Comment:
Should use `'OneSignalXCFramework'` instead of `'OneSignal'`, as the latter does not support the iOS simulator on ARM macs (M1).

Reply:
Good catch